backend/app/modules/searchengine/SearchActor.scala

```diff
@@ -53,6 +59,14 @@ class SearchActor (monitoring: ActorRef) extends Actor {
 
     val currentSender = sender
 
+    val messageInLowerCase = message.toLowerCase
+
+    val event = sparql
+      .ask(FetchEventClasses())
+      .mapTo[SparQLFetchResponse]
```

Comment:
SparQLFetchResponse doesn't exist anymore, need to be a SparQLEventClassesResponse